demos-europe/demosplan-core
# feat: make split statement button in list statements available again

- Basically this reverts the changes of this commit #b9ffea39025c8de05645060b10b9fd90e26db75b
- Since split statements is addonized now, we need to make the button in the list statement view available again

### How to review/test

- Login as FPA
- go to list statement view
- make sure unsegmented statements can be segmented via the flyout button
- if already has segments button should be disabled
